### PR TITLE
py-bcsb: release 2.4.1.

### PR DESCRIPTION
**Pull request by Adrien4193**

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/py-bcsb/package.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class PyBcsb(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/viz/brayns/braynscircuitstudiobackend.git"
 
     version("develop", branch="develop")
-    version("2.4.0", tag="v2.4.0")
+    version("2.4.1", tag="v2.4.1")
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-bcsb/package.py:16]: That tag does not seem to exist.

**Review by matz-e — CHANGES_REQUESTED** [review]

File: bluebrain/repo-bluebrain/packages/py-bcsb/package.py (hunk 1 above)
Comment (Adrien4193) [reply bluebrain/repo-bluebrain/packages/py-bcsb/package.py:16]: My bad ^^'

Comment (Adrien4193) [reply bluebrain/repo-bluebrain/packages/py-bcsb/package.py:16]: Could you retrigger the CI now that I created the tag?

Comment (Adrien4193) [reply bluebrain/repo-bluebrain/packages/py-bcsb/package.py:16]: Fixed thanks.

Just need you blessing now ;)

**Review by matz-e — APPROVED** [review]

File: bluebrain/repo-bluebrain/packages/py-bcsb/package.py (hunk 1 above)
Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/py-bcsb/package.py:16]: There you go :)

**Pull request merged**